src/gap.c

```diff
@@ -876,6 +876,31 @@ Obj FuncRUNTIMES( Obj     self)
 }
 
 
+
+/****************************************************************************
+**
+*F  FuncGetTimeOfDay( <self> )
+**
+**  'FuncGetTimeOfDay' implements the function 'GetTimeOfDay'.
+**
+**  'GetTimeOfDay' returns the current time since 1.1.1970 in seconds and
+**  microseconds as per the POSIX specification.
+**  The accuracy of this is system dependent.
+** 
+**  Also note that gettimeofday has been marked obsolete in the POSIX
+**  standard.
```

Comment:
This then raises the question why we are using it -- I think this question should be antiticpated, and thus answered here.